awx/ui_next/src/components/ResourceAccessList/__snapshots__/DeleteRoleConfirmationModal.test.jsx.snap

```diff
@@ -384,7 +365,6 @@ exports[`<DeleteRoleConfirmationModal /> should render initially 1`] = `
               isOpen={true}
               labelId="pf-modal-part-1"
               onClose={[Function]}
-              ouiaSafe={true}
```

Comment:
Why remove safe?

Reply:
Hey @unlikelyzero, I am taking a look into those changes again.

Reply:
hey @unlikelyzero it was a weird snapshot update. I rebuilt my env, and the updated snapshot changes seem to be more align with the initial change.